src/main/kotlin/com/ampnet/payoutservice/controller/response/PayoutResponse.kt

```diff
@@ -0,0 +1,35 @@
+package com.ampnet.payoutservice.controller.response
+
+import com.ampnet.payoutservice.util.PayoutStatus
+import com.fasterxml.jackson.databind.annotation.JsonSerialize
+import com.fasterxml.jackson.databind.ser.std.ToStringSerializer
+import java.math.BigInteger
+import java.util.UUID
+
+data class PayoutResponse(
+    val taskId: UUID?,
+    val status: PayoutStatus,
+    val issuer: String?,
+
+    @JsonSerialize(using = ToStringSerializer::class)
+    val payoutId: BigInteger?,
+    val payoutOwner: String,
+    val payoutInfo: String?,
+    val isCanceled: Boolean?,
+
+    val asset: String,
+    @JsonSerialize(using = ToStringSerializer::class)
+    val totalAssetAmount: BigInteger?,
+    val ignoredAssetAddresses: Set<String>,
+
+    val assetSnapshotMerkleRoot: String?,
+    val assetSnapshotMerkleDepth: Int?,
+    @JsonSerialize(using = ToStringSerializer::class)
+    val assetSnapshotBlockNumber: BigInteger,
+
```

Comment:
Just to check, in the task, payout response has `string assetSnapshotMerkleIpfsHash`, why is it missing here? https://linear.app/ampnet/issue/SD-572/add-endpoints-to-fetch-all-payouts-per-issuer-and-investor

Reply:
I forgot to add it to the response, but I saw it was missing when writing integration tests. It is in the response now.